### PR TITLE
Fixes typos mistakes and errors

### PR DESCRIPTION
**Pull request by theranjitraut**
More than dozens of typos and mistakes are fixed

### CODE REVIEW

File: files/en-us/mdn/contribute/open_source_etiquette/index.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ Always find out where the best place is to ask questions. Good OSPs will always
 
 ## Make progress, not noise
 
-Think carefully about the way you handle communication in the project — make sure it is useful, and that it doesn't make other contributor's jobs harder. Submitting pull requests to fix bugs is great, but are they truly useful, and easy to review? Filing issues and joining in other conversations is fine, but are your issues and comments on topic, or are they just adding noise?
+Think carefully about the way you handle communication in the project — make sure it is useful, and that it doesn't make other contributors' jobs harder. Submitting pull requests to fix bugs is great, but are they truly useful, and easy to review? Filing issues and joining in other conversations are fine, but are your issues and comments on topic, or are they just adding noise?
```
Comment (dipikabh) [inline files/en-us/mdn/contribute/open_source_etiquette/index.md:107]:
```suggestion
Think carefully about the way you handle communication in the project — make sure it is useful, and that it doesn't make other contributors' jobs harder. Submitting pull requests to fix bugs is great, but are they truly useful, and easy to review? Filing issues and joining in other conversations is fine, but are your issues and comments on topic, or are they just adding noise?
```

The original sentence is correct. It's like saying 'doing A and B is fine'

File: files/en-us/mdn/contribute/open_source_etiquette/index.md [hunk 2]
```diff
@@ -10,60 +10,60 @@ tags:
...
 
 Even better reasons might include:
 
 - I use this tool all the time and found a bug in it/want to help make it better.
 - I want to help other people use the tool more successfully.
 - I want to help other people contribute to the project more successfully.
-- I want to improve my own skills.
-- I want to publicly demonstrate my own skills as part of my college or university course.
-- I want to publicly demonstrate my own skills to improve my chances of getting a job.
+- I want to improve own skills.
```
Comment (dipikabh) [inline files/en-us/mdn/contribute/open_source_etiquette/index.md:26]:
```suggestion
- I want to improve my skills.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks @theranjitraut for the edits. I have suggested a few changes.

File: files/en-us/mdn/contribute/open_source_etiquette/index.md [hunk 3]
```diff
@@ -10,60 +10,60 @@ tags:
...
-## Be polite, be kind, avoid incendiary or offensive language
+## Be polite, be kind, and avoid incendiary or offensive language
 
 We could abbreviate this to "be kind". This is our number one bit of advice for anyone starting open source contributions.
 
 Be kind to the other contributors on the project, and it will be a happier and more productive place. This includes:
 
 - Thanking people if they help you.
-- Congratulating people where appropriate (for example if they land their first ever pull request, or fix a particularly difficult bug).
+- Congratulating people where appropriate (for example, if they land their first-ever pull request, or fix a particularly difficult bug).
```
Comment (dipikabh) [inline files/en-us/mdn/contribute/open_source_etiquette/index.md:47]:
```suggestion
- Congratulating people where appropriate (for example, when they land their first pull request or fix a particularly difficult bug).
```

Comment (dipikabh) [reply files/en-us/mdn/contribute/open_source_etiquette/index.md:47]: Sorry @theranjitraut, I added one more suggested change before I saw your fixes

**Review by dipikabh — APPROVED** [review]
Thanks, @theranjitraut!

**Comment by theranjitraut** [comment]
You're Ma'am

**Pull request merged**